source/apiVolontaria/volunteer/templates/calendar_invitation.txt

```diff
@@ -0,0 +1,16 @@
+Récupérer le calendrier de vos participations à nos évènements
+--------------------------------------------------------------
```

Comment:
Add I18N to source/apiVolontaria/volunteer/templates/calendar_invitation.txt

(third time :p )